governance/polls/Nexo Institutional Vaults Onboarding - October 11, 2021.md

```diff
@@ -0,0 +1,116 @@
+   1: Yes
+   2: No
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Poll: Nexo Institutional Vault Onboarding - October 11, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Growth Core Unit. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, October 11 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This proposal covers onboarding two new vault types. For each vault type there is a set of initial parameters. Additionally, there are a set of conditions which both Nexo and MakerDAO agree to operate in relation to these vaults if this vote passes successfully. 
+
+### INST-ETH-A - Starting Parameters
+Starting Stability Fee: 1.5%
+Origination Fee: 0%
+Liquidation Ratio: 120%
+Minimum Active CR: 150%
+Debt Ceiling: 900m
+Cut: 0.99
+Step: 90 seconds
+Buf: 1.20
+Cusp: 0.4
+Tail: 140 minutes
+Chip: 0.1%
+Tip: 300 DAI
+lk.chop: 20%
+Tolerance: 0.5
+Ilk.hole: 50m DAI
+Dust: 10,000 DAI
+
+### INST-WBTC-A - Starting Parameters
+Starting Stability Fee: 1.5%
+Origination Fee: 0%
+Liquidation Ratio: 120%
+Minimum Active CR: 150%
+Debt Ceiling: 600m
+Cut: 0.99
+Step: 90 seconds
+Buf: 1.20
+Cusp: 0.4
+Tail: 140 minutes
+Chip: 0.1%
+Tip: 300 DAI
+Ilk.chop: 20%
+Tolerance: 0.5
+Ilk.hole: 30m DAI
+Dust: 10,000 DAI
+
+### Operating Conditions
+
+#### Migration
+* MakerDAO will facilitate Nexo in migrating their existing positions in the ETH-A and WBTC-A vault types to these two new vault types.
+* Nexo will be unable to utilize either of these vault types until they have migrated a total of 400M DAI worth of debt across both vaults.
+* A 1% origination fee will be charged on the migrated amount above 400M DAI.
+* Once migration has been completed, the following parameters will be set for each vault. These parameters may be freely changed in the future by MakerDAO.
+	* Origination Fee: 1%
+	* DC-IAM gap: 50m
+	* DC-IAM ttl: 8h
+	* DC-IAM line: Equal to initial Debt Ceiling
+
+#### First Six Months
+* Every additional 100m DAI minted in each vault will reduce the stability fee by 10 basis points for that vault down to a minimum of 0.5%. The starting point for this calculation is the amount of DAI initially migrated by MakerDAO on behalf of Nexo.
+* The Stability Fee will not change in the initial six months with the exception of reductions under the previous point.
+* Reductions in Stability Fee are not automatic, and must be voted through by Maker Governance. The Growth Core Unit is responsible for initiating reduction proposals on behalf of Nexo.
+
+#### Subsequent Six Months
+* The Growth Core Unit will facilitate discussions to determine parameters for the subsequent six months prior to the expiry date.
```

Comment:
I dont remember this loan having expiration date?

Reply:
Yeah, I can clarify.